lyluatex.lua

```diff
@@ -129,13 +131,11 @@ end
 
 
 local function locate(file, includepaths, ext)
-    local result = ly.CURRFILEDIR..file
-    if not lfs.isfile(result) then result = file end
-    if not lfs.isfile(result) then
-        for _, d in ipairs(extract_includepaths(includepaths)) do
-            result = d..'/'..file
-            if lfs.isfile(result) then break end
-        end
+    local result
+    for _, d in ipairs(extract_includepaths(includepaths)) do
+        if d:sub(-1) ~= '/' then d = d..'/' end
+        result = d..file
+        if lfs.isfile(result) then break end
     end
     if not lfs.isfile(result) then result = kpse.find_file(file) end
     if not result and ext and file:match('.%w+$') ~= ext then return locate(file..ext, includepaths) end
```

Comment:
@jperon It is unrelated to the PR but I don't see what you want to achieve with the match here. If I read this correctly it will match "any character, followed by one or more alphanumerical characters" which will never match the extension but something at the beginning. Shouldn't that rather be `match('%..+)` to match a literal dot followed by any (and not only alphanumerical) characters?

Reply:
Right; fixed in #123. It worked as it was (because the `.` wasn't followed by `+`, `*` or `?`, but it's more readable now.